style.css

```diff
@@ -0,0 +1,75 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@400;700&display=swap');
+
+body {
+  width: 375px;
+}
+
+.main {
```

Comment:
- [ ] As mentioned in the requirement (screenshot below), kindly use display flex on this class to align the items inside the container
![image](https://user-images.githubusercontent.com/73140461/143473560-c8d255bc-6385-413e-8798-f65a0c9cd711.png)